utils/swift_build_sdk_interfaces.py

```diff
@@ -410,18 +410,9 @@ def main():
     # Copy a file containing SDK build version into the prebuilt module dir,
     # so we can keep track of the SDK version we built from.
     copySystemVersionFile(args.sdk, args.output_dir)
-    if 'ANDROID_DATA' not in os.environ:
-        shared_output_lock = multiprocessing.Lock()
-        pool = multiprocessing.Pool(args.jobs, set_up_child,
-                                    (args, shared_output_lock))
-    else:
-        # Android doesn't support Python's multiprocessing as it doesn't have
-        # sem_open, so switch to a ThreadPool instead.
-        import threading
-        shared_output_lock = threading.Lock()
-        from multiprocessing.pool import ThreadPool
-        pool = ThreadPool(args.jobs, set_up_child,
-                          (args, shared_output_lock))
+    shared_output_lock = multiprocessing.Lock()
+    pool = multiprocessing.Pool(args.jobs, set_up_child,
+                                (args, shared_output_lock))
```

Comment:
Someone recently ported `sem_open` to Termux, termux/termux-packages#8993, so go the other way and remove the Android-specific code instead, now that this works.